### DISCUSSION TITLE
Welcome to DS4Windows Discussions!

### DISCUSSION THREAD

**Discussion by nefarius** [CircumSpector/DS4Windows#20] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by matsher** [comment]
Hi DS4Win Fam. I have been experimenting with the software and its great - lots I still need to learn about it.
I have a question I hope can be answered.
Can the ds4Win software zero the DS4 sensors at a different angle/attitude than the factory default???
Kinda like a simple spirit-level app on your cell phone or like the track ir can... You can press a button and 
the sensors will all zero/centre at whatever angle you've positioned the device.

It seems the ds4 app can do pretty much everything else to remap the controllers.
Can anyone tell me if this can be done.
Thanks so much.

**Reply by nefarius** [reply]
That is not yet implemented, food for thought for the future.
